### PR TITLE
Update psud HLD

### PR DESCRIPTION
**Pull request by orfar1994**
Update psud HLD with the new schema of PSU_INFO.

Currently psud collect the psu's output voltage and current, we would like to collect the psu's input voltage and current.
Psud collect it in the same way it collect the output voltage and current (by calling to psu lib and for Mellanox psu lib access to hw-mgmt sysfs - "/var/run/hw-management/")

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@stephenxs could you please help to review?

File: doc/psud/PSU_daemon_design.md [hunk 1]
```diff
@@ -4,10 +4,13 @@
 
 ### Revision ###
 
+
  | Rev |     Date    |       Author       | Change Description                |
  |:---:|:-----------:|:------------------:|-----------------------------------|
  | 0.1 |             |      Chen Junchao  | Initial version                   |
  | 0.2 | August 4st, 2022 | Stephen Sun | Update according to the current implementation |
+ | 0.3 | August 8st, 2022 | Or Farfara | Add input current, voltage and max power |
```
Comment (stephenxs) [inline doc/psud/PSU_daemon_design.md:12]: `8th`. typo. can you also fix the typo `4st` in line 11? thank you.

File: doc/psud/PSU_daemon_design.md [hunk 2]
```diff
@@ -58,7 +61,10 @@ PSU information is stored in PSU table:
     voltage_max_threshold   = 1*3.3DIGIT                     ; the maximum voltage threshold of the PSU
     current                 = 1*3.3DIGIT                     ; the current of the PSU
     power                   = 1*3.3DIGIT                     ; the power of the PSU
-
+	  input_voltage           = 1*3.3DIGIT                     ; input voltage of the psu
+	  input_current           = 1*3.3DIGIT                     ; input current of the psu
+	  max_power               = 1*3.3DIGIT                     ; power capacity of the psu
```
Comment (stephenxs) [inline doc/psud/PSU_daemon_design.md:66]: maybe `4.3DIGIT` for max power? I think it's possible for a PSU to provide a power greater than 1kw? how do you think?

File: doc/psud/PSU_daemon_design.md (hunk 1 above)
Comment (orfar1994) [reply doc/psud/PSU_daemon_design.md]: Sure,done

File: doc/psud/PSU_daemon_design.md (hunk 2 above)
Comment (orfar1994) [reply doc/psud/PSU_daemon_design.md]: You right, done

**Review by stephenxs — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Review by liat-grozovik — CHANGES_REQUESTED** [review]
@orfar1994 please explain also the motivation of adding that and what is the logic behind it.
how will you get this info into psu daemon?

**Comment by orfar1994** [comment]
Currently psud collect the psu's output voltage and current, we would like to collect the psu's input voltage and current.
Psud collect it in the same way it collect the output voltage and current (by calling to psu lib and for Mellanox psu lib access to hw-mgmt sysfs - "/var/run/hw-management/")

**Comment by liat-grozovik** [comment]
@orfar1994 could you please update the PR description/HLD with the logic added? then i can merge it

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**